saleor/checkout/tests/test_calculations.py

```diff
@@ -521,3 +522,87 @@ def test_fetch_checkout_prices_when_tax_exemption_and_not_include_taxes_in_price
 
     assert checkout.total == shipping_price + all_lines_total_price
     assert checkout.subtotal == all_lines_total_price
+
+
+@freeze_time()
+@patch("saleor.plugins.avatax.plugin.AvataxPlugin")
```

Comment:
The test should not be strictly related to AvataxPlugin. If you want to test single tax plugin call, you should mock PluginsManager, and confirm that the request was passed to single plugin (or just confirm that proper method for calling the method on single plugin was called)

Reply:
Tests have own dummy plugins, which can be used: `saleor/plugins/tests/sample_plugins.py`